### DISCUSSION TITLE
Remove octoapp companion fail

### DISCUSSION THREAD

**Discussion by ankokdog** [Guilouz/Creality-Helper-Script-Wiki#719] [Issues]
hi. i have installed the helper script to a ki max and also install the octoapp companion. But i want to remove it. I am doing the uninstall procedure but when i go to information menu it soes that it is still there. i have tried many times to uninstall the procedure says at the end

You're about to uninstall OctoApp.
This printer ID will be deleted, but you can always reinstall the plugin and re-add this printer.

Are you want to uninstall? [y/n]Y

Starting OctoApp uninstall
No local plugins or companions were found to remove.

 ✓ OctoApp Companion has been removed successfully!

every time i go to unistall it starts the procedure. when i go to some other part to uninstall and had been uninstalles in the past it says that ✗ OctoEverywhere is not installed! for example, but with octoprint it always start the uninstall procedure

**Comment by Unwrapped3** [comment]
Some issue on 3KE. The script says that the companion app had been removed successfully. However, when trying to reinstall, the script says that the companion app is already installed. Any help would be appreciated.
